### DISCUSSION TITLE
Enable3D error "Object3D is not defined"

### DISCUSSION THREAD

**Discussion by Aditya8697** [enable3d/enable3d#310] [Help / Questions]
How do i setup enabled3d I downloaded the "**enable3d.ammoPhysics.0.25.2.min.js**" file and mentioned it in my html just like the examples but it dsnt seem to work, these are the errors : 
![Screenshot_20220804-125447_Spck Editor](https://user-images.githubusercontent.com/109073576/182789999-179f83f0-649b-4c3d-af02-7555bb33f9db.jpg)
And here is my code where I linked the enable3d file
![Screenshot_20220804-125457_Spck Editor](https://user-images.githubusercontent.com/109073576/182790201-1e2e1d37-4e61-4e42-961e-76798b9a3524.jpg)
I am new with enable3d and i am really confused. How to setup it.

**Comment by vitalspace** [comment]
check https://enable3d.io/examples/native-three-with-physics.html
